### ISSUE TITLE
How to pass a parameter to loss function?

### ISSUE THREAD

**Issue by nvidiaman** [asyml/texar#138]
Hi,

I need to implement a loss function to which i can pass a parameter value at each epoch.

i.e. loss(input, output, epoch)

Is it possible in texar i.e. how to pass a paremeter to the loss function at each epoch?

a simple snippet demonstrating this in a _train loop will be very helpful. thank you!

**Comment by ZhitingHu** [comment]
Not sure what you meant by "pass a parameter to loss". did you mean sth like
```
def loss(input, output, epoch):
   pred = model(input)
   return loss_fn(pred, output)
```

**Comment by nvidiaman** [comment]
@ZhitingHu thanks for the reply. 

yes, I mean that the loss function will have the epoch or as a parameter, so in each epoch i will compute loss like: loss(input, output, epoch).

since, in tensorflow the model is built first and then the session starts, any simple example demonstrating this in texar will be very helpful

**Comment by ZhitingHu** [comment]
you may set epoch as a placeholder, and feed in the actual epoch during session run. As a reference, check out this [line](https://github.com/asyml/texar/blob/master/examples/language_model_ptb/lm_ptb.py#L109) where we set global_step as a placeholder.
